native_client/client.cc

```diff
@@ -12,7 +12,11 @@
 #include <sstream>
 #include <string>
 
-#if defined(__ANDROID__) || defined(_MSC_VER)
+#ifdef __APPLE__
+#include <TargetConditionals.h>
+#endif
+
+#if defined(__ANDROID__) || defined(_MSC_VER) || TARGET_OS_IPHONE
```

Comment:
really `TARGET_OS_IPHONE` ? Shouldn't it be `defined()` somehow? Also, do you know if that covers only iPhone or any iOS hw ?

Reply:
These variables are booleans, they're always defined in `TargetConditionals.h`. We could check `#if defined(TARGET_OS_IPHONE) && TARGET_OS_IPHONE` but that's redundant:

```
$ cat -n test.cpp
     1	#include <stdio.h>
     2
     3	int main(void) {
     4	#if FOOBAR
     5	    printf("FOOBAR!\n");
     6	#endif
     7	    return 0;
     8	}
$ clang -DFOOBAR=0 test.cpp && ./a.out
$ clang -DFOOBAR=1 test.cpp && ./a.out
FOOBAR!
$ clang test.cpp && ./a.out
$
```

Reply:
Ok, it just looks weird that way, but if you say so :)